src/content/resources/glossary.md

```diff
@@ -132,6 +132,15 @@ For additional details, see the
 
 [definiteAssignmentSpec]: https://github.com/dart-lang/language/blob/main/resources/type-system/flow-analysis.md
 
+## Function
+
+"Function" refers to top-level functions, local functions, static
```

Comment:
Sorry I wasn't paying attention to the final change during the GVC 😬 but I actually like the way you had it at first!

```suggestion
Unless noted otherwise, the term *function* refers to top-level functions, local functions, static
```

The change I was referring to was changing "function" (in quotes) to *function* (italicized) to match the other entries. Thanks!